vault/logical_system_helpers.go

```diff
@@ -91,6 +92,10 @@ var (
 		return nil
 	}
 
+	handlePluginReloadStatus = func(ctx context.Context, store logical.Storage, req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
```

Comment:
Since the args aren't used...

```suggestion
	handlePluginReloadStatus = func(_ context.Context, _ logical.Storage, req *logical.Request, _ *framework.FieldData) (*logical.Response, error) {
```

Reply:
TIL